packages/http/src/router/__tests__/utils.ts

```diff
@@ -30,19 +40,15 @@ interface IRandomPathOptions {
 }
 
 export function randomPath(opts: IRandomPathOptions = defaultRandomPathOptions): string {
-  opts = Object.assign({}, defaultRandomPathOptions, opts);
+  defaults(opts, defaultRandomPathOptions);
```

Comment:
Nice, I didn't know `defaults`. I'm not a huge fan of mutating arguments, though. Is this not a problem?

Reply:
Me neither, but you were mutating them anyway in the previous statement.

`opts` is passed externally as an argument — and then they get overridden with `opts = Object.assign({…})`

Reply:
f671266